examples/.babelrc

```diff
@@ -1,2 +1,3 @@
 {
+    "presets": ["../babel"]
```

Comment:
Earlier our location detection logic couldn't find this.
So, we add `next/babel` by default. 

That's why providing a empty `.babelrc` here worked. 
But now we have to do this.